### ISSUE TITLE
Visual indication if a filter has been applied to the transaction log

### ISSUE THREAD

**Issue by orhoj** [Concordium/concordium-desktop-wallet#146]
**Description**
If a transaction filter is applied to the transaction log, then some transactions may be hidden. It is not obvious, as a user, whether a filter has been applied or not. This can only be discovered by navigating to the Filter page to check the current settings. A visual indicator (like an icon) would be an improvement, so that it becomes clear that the full list of transactions is not currently shown.

**Steps to Reproduce**
Change view -> Filters -> Filter out some transaction types that you have in your transaction log.

**Versions**
1.2.0

**Comment by orhoj** [comment]
@jens-concordium I can see that the UI was changed regarding "change view", but did you implement anything with regards to this issue?

**Comment by jens-concordium** [comment]
I don't think we made any changes for this yet. I think we discussed it without finding a solution, didn't we @soerenbf?
